src/app/examples/pos/pos.html

```diff
@@ -47,10 +46,7 @@
   <img
     style="margin-top: 10px"
     class="responsive-img"
-    [src]="
-      (service.meow$ | async | successValue)?.file ||
-      (service.meow$ | async | inProgressValue)?.file
-    "
+    [src]="(service.meow$ | async | successValue)?.file"
```

Comment:
And if you agree and we show this section only when it's success, we won't need this check for null `?` anymore

Reply:
The `?` is needed because the `successValue` pipe has some logic for providing a `defaultValue` that allows the transform to return `undefined` (because `defaultValue` is optional.